layouts/partials/site-scripts.html

```diff
@@ -1,5 +1,5 @@
 <script src="{{ `js/app.js` | absURL }}"></script>
-<script src="{{ `js/playground-search.js` | absURL }}"></script>
+<!-- <script src="{{ `js/playground-search.js` | absURL }}"></script> -->
```

Comment:
I noticed an error in the console related to this script and thought it would be best to hide this script for now since it is not currently in use.